esp-hal/src/timer.rs

```diff
@@ -156,7 +157,8 @@ impl TimerGroupInstance for TIMG1 {
     fn configure_src_clk() {
         unsafe {
             (*Self::register_block())
-                .t1config()
```

Comment:
I'm not familiar with ESP timers, but is this correct?

Reply:
Looks okay to me, but I will do some testing on my end too

Reply:
TIMG0 & TIMG1 still work correctly on a esp32s2 and esp32s3 board I tested here

Reply:
should there be t(0).config for both TIMG0 and TIMG1?

Reply:
![image](https://github.com/esp-rs/esp-hal/assets/6977289/8cff877f-fa5e-4a0e-b4d3-4b08376626cd)

Huh, yeah I think you're right! It seems each timer in each timer group can select its clock source individually, I will create an issue. Thanks!